infrastructure/variables.tf

```diff
@@ -10,12 +10,64 @@ variable "location" {
   default = "UK South"
 }
 variable "env" {}
+
 variable "subscription" {
   default = ""
 }
+
 variable "deployment_namespace" {
   default = ""
 }
+
 variable "common_tags" {
   type = map(string)
 }
+/*
+# Resource Group variables
```

Comment:
Probably don't need these if all commented out?

Reply:
Thanks @jackmaloney, as above, I'll remove everything that isn't necessary in a code clean up when we have something working.